### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by snyder114**
Basic formatting and wording enhancements for readability and accessibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,137 +83,135 @@ java -javaagent:path/to/opentelemetry-javaagent-all.jar \
...
-| otel.exporter.jaeger.service.name | OTEL_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance, default is "unknown".                                       |
+| otel.exporter=jaeger              | OTEL_EXPORTER=jaeger              | Select the Jaeger exporter                                                                          |
+| otel.exporter.jaeger.endpoint     | OTEL_EXPORTER_JAEGER_ENDPOINT     | The Jaeger endpoint to connect to. Default is `localhost:14250`. Currently only gRPC is supported. |
+| otel.exporter.jaeger.service.name | OTEL_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance. Default is `unknown`.                                       |
 
 ##### Zipkin exporter
-A simple wrapper for the Zipkin exporter of opentelemetry-java. It POSTs json in [Zipkin format](https://zipkin.io/zipkin-api/#/default/post_spans) to a specified HTTP URL.
+A simple wrapper for the Zipkin exporter of opentelemetry-java. It posts JSON in [Zipkin format](https://zipkin.io/zipkin-api/#/default/post_spans) to a specified HTTP URL.
```
Comment (trask) [inline README.md:122]:
```suggestion
A simple wrapper for the Zipkin exporter of opentelemetry-java. It sends JSON in [Zipkin format](https://zipkin.io/zipkin-api/#/default/post_spans) to a specified HTTP URL.
```

**Review by trask — APPROVED** [review]
Thanks @snyder114, these are great improvements!

**Review by iNikem — APPROVED** [review]
Thank you!

And @trask , we should split our README :)

**Comment by snyder114** [comment]
@iNikem @trask happy to help! Could one of you merge the PR for me? I'm not authorized.

**Comment by trask** [comment]
Closing and re-opening to re-trigger CLA check

**Comment by trask** [comment]
Trying again to re-trigger CLA check

**Comment by anuraaga** [comment]
Thanks!

**Comment by snyder114** [comment]
Thanks for your help everyone!

**Pull request merged**